manifests/operator-service-account-rbac.yaml

```diff
@@ -142,7 +142,16 @@ rules:
   - bind
   resourceNames:
   - zalando-postgres-operator
-
+- apiGroups:
+  - batch
+  resources:
+  - cronjobs # enables logical backups
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
```

Comment:
update, too?

Reply:
Remember to add this in the helm chart template as well. Same for other parameters.

Reply:
> Remember to add this in the helm chart template as well. 

i've already them to `values.yaml`

Reply:
referring to [this](https://github.com/zalando/postgres-operator/blob/master/charts/postgres-operator/templates/clusterrole.yaml) template file